cider/exec/nextgen/jitlib/base/JITModule.h

```diff
@@ -30,6 +30,10 @@ class JITModule {
       const JITFunctionDescriptor& descriptor) = 0;
 
   virtual void finish() = 0;
+  virtual void addExternalModules(JITModule& external_module) = 0;
+
+ protected:
+  JITModule* external_module_;
```

Comment:
1. should we maintain `external_module_`?  if yes, `addExternalModules` is not safe.
2. should this be a vector for multiple module?

Reply:
now change maintain a ```external_module``` instance instead of a reference, so ```addExternalModules``` has been deleted.
explanation about multiple module is in the above conversation